### ISSUE TITLE
"pip install" is broken on today's release

### ISSUE THREAD

**Issue by Akuli** [Akuli/porcupine#1184]
Running the installation command from README gives:
```
$ python3 -m pip install https://github.com/Akuli/porcupine/archive/v2022.08.27.zip
Collecting https://github.com/Akuli/porcupine/archive/v2022.08.27.zip
  Using cached https://github.com/Akuli/porcupine/archive/v2022.08.27.zip
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
ERROR: Invalid requirement: 'tree-sitter@ https://github.com/Akuli/py-tree-sitter-builds/releases/download/v2022-08-27/tree_sitter-0.20.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl; python_version == "3.9" and sys_platform == "linux" and platform_machine == "x86_64"'
```

**Comment by Akuli** [comment]
Another problem with today's release is that the `yml` files didn't get copied. I hate setuptools.

**Comment by Akuli** [comment]
Fixed by #1187 and #1188

**Comment by Akuli** [comment]
Released now
